dex/farm-staking/src/lib.rs

```diff
@@ -75,10 +77,7 @@ pub trait Farm:
             farming_token_id != farm_token,
             "Farming token ID cannot be farm token ID"
         );
-        require!(
-            max_apr > 0 && max_apr < MAX_PERCENT,
-            "Invalid max APR percentage"
-        );
+        require!(max_apr > 0, "Invalid max APR percentage");
 
         self.state().set(&State::Inactive);
         self.penalty_percent()
```

Comment:
delete this call. not needed. also the function if it is not in common. otherwise that can stay.